### PR TITLE
Issue #22 multiple test cases

### PR DESCRIPTION
**Pull request by flbulgarelli**

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.1.0
+2.0.0-p481
```
Comment (faloi) [inline .ruby-version]: Why is this necessary?

File: lib/gobstones/spec/example.rb [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  class Example
+    include Gobstones::WithTempfile
+
+    attr_reader :gobstones_path
+
+    def initialize(gobstones_path)
+      @gobstones_path = gobstones_path
+    end
+
+    def start!(source_file, initial_board, final_board)
```
Comment (faloi) [inline lib/gobstones/spec/example.rb]: I still don't get where to use the bang (!) and where not... can you shed some light on this?

File: spec/integration_spec.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  let(:bridge) { Mumukit::Bridge::Bridge.new('http://localhost:4567') }
+
+  before(:all) do
+    @pid = Process.spawn 'RACK_ENV=development rackup -p 4567'
+    sleep 3
+  end
+  after(:all) { Process.kill 'QUIT', @pid }
+
+  context 'when submission is ok' do
+    it 'answers a valid hash' do
```
Comment (faloi) [inline spec/integration_spec.rb]: I don't see a lot of value on these tests... perhaps I'm missing something.

Comment (flbulgarelli) [inline spec/integration_spec.rb]: It is an integration test that validate that test runner, test compiler and expectation runner properly work together.

File: lib/gobstones/spec/example.rb (hunk 2 above)
Comment (flbulgarelli) [inline lib/gobstones/spec/example.rb]: Ruby convention: if it has some kind of effect, it is a good idea to signal it that way. 

However, not every ruby code follows this convention.
